packages/run-protocol/src/vpool-xyk-amm/addLiquidity.js

```diff
@@ -25,9 +26,9 @@ const makeMakeAddLiquidityInvitation = (zcf, getPool) => {
 
     // Get the brand of the secondary token so we can identify the liquidity pool.
     const secondaryBrand = seat.getProposal().give.Secondary.brand;
-
     const pool = getPool(secondaryBrand);
-    const liquidityBrand = pool.getLiquidityIssuer().getBrand();
+    const liquidityIssuer = E(pool).getLiquidityIssuer();
+    const liquidityBrand = await E(liquidityIssuer).getBrand();
```

Comment:
This is not a good reason for making the offer hook async and have an `await`.  There should be an API that provides the brand from the ZCF, which should be synchronous.

Reply:
If you have the issuer inside a contract, you can directly call `zcf.getBrandForIssuer(issuer)` to get the brand synchronously.

Reply:
Just got that working, thanks. PTAL.